package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "tracetoio-crypto-lib",
-  "version": "1.1.1",
+  "version": "1.2.2",
```

Comment:
This seems to break if the parameters are different?

Reply:
updated